### PR TITLE
fix(how-to-upgrade): Change "Prisma 2" to "Prisma version 2.x and later"

### PR DESCRIPTION
**Pull request by janpio**

### CODE REVIEW

File: content/300-guides/300-upgrade-guides/800-upgrade-from-prisma-1/01-how-to-upgrade.mdx [hunk 1]
```diff
@@ -28,37 +28,38 @@ Once you're done with that guide, you can choose **one of the following four gui
...
-- Prisma 2 uses its own [modeling language](https://github.com/prisma/specs/tree/master/schema) instead of being based on GraphQL SDL.
-- Prisma 2 doesn't expose ["a GraphQL API for your database"](https://www.prisma.io/blog/prisma-and-graphql-mfl5y2r7t49c/) anymore, but only allows for _programmatic access_ via the Prisma Client API. This means Prisma 2 [doesn't support Prisma binding](https://www.prisma.io/docs/more/faq#does-prisma-client-support-graphql-schema-delegation-and-graphql-binding) any more.
-- More powerful introspection allows connecting Prisma 2 to any existing database
+- Prisma 2+ uses its own [modeling language](https://github.com/prisma/specs/tree/master/schema) instead of being based on GraphQL SDL.
+- Prisma 2+ doesn't expose ["a GraphQL API for your database"](https://www.prisma.io/blog/prisma-and-graphql-mfl5y2r7t49c/) anymore, but only allows for _programmatic access_ via the Prisma Client API. 
+  - This means Prisma 2+ [doesn't support Prisma binding](https://www.prisma.io/docs/more/faq#does-prisma-client-support-graphql-schema-delegation-and-graphql-binding) any more.
```
Comment (janpio) [inline content/300-guides/300-upgrade-guides/800-upgrade-from-prisma-1/01-how-to-upgrade.mdx]: Unrelated change to make this bullet point separate from the more general main bullet.

File: content/300-guides/300-upgrade-guides/800-upgrade-from-prisma-1/01-how-to-upgrade.mdx [hunk 2]
```diff
@@ -108,29 +109,29 @@ Note that the Upgrade CLI is designed in a way that **you can stop and re-start
...
 
-Note that you won't be able to fix the [schema incompatibilities](#schema-incompatibilities) that require are breaking Prisma 1 if you choose the gradual upgrade strategy and intend to run Prisma 1 and Prisma 2 side-by-side. That's because these data migrations are breaking the schema that Prisma 1 expects. This means that your Prisma Client API might not feel as idiomatic as it could, but you still get the full feature set of Prisma Client.
+Note that you won't be able to fix the [schema incompatibilities](#schema-incompatibilities) that require a "Breaking Prisma 1" change if you choose the gradual upgrade strategy and intend to run Prisma 1 and Prisma 2+ side-by-side. That's because these data migrations are breaking the schema that Prisma 1 expects. This means that your Prisma Client API might not feel as idiomatic as it could, but you still get the full feature set of Prisma Client.
```
Comment (janpio) [inline content/300-guides/300-upgrade-guides/800-upgrade-from-prisma-1/01-how-to-upgrade.mdx]: Unrelated fix to make the sentence to make a bit more sense. Still not optimal, but should be better at least.

**Review by nikolasburk — APPROVED** [review]
Looks great, thank you Jan! 🙏

**Comment by tanberry** [comment]
Hi @janpio I replaced references to "Prisma 2+" with "Prisma version 2.x and later" (or some variation thereof). It's a little longer, and perhaps more clunky, but doesn't raise the question of what 2+ means.

**Comment by janpio** [comment]
Not great, but fine with me. (Not that other pages still might need to be adapted as well) (Please also update the PR title to the chosen term before merging)

**Comment by janpio** [comment]
Ok, this looks good to me - let's just merge and you take care of the other pages independently @tanberry

**Review by tanberry — APPROVED** [review]
